C2GUILauncher/src/Views/MainWindow.xaml

```diff
@@ -129,16 +129,18 @@
                 </Grid.RowDefinitions>
 
                 <Grid.ColumnDefinitions>
-                    <ColumnDefinition/>
+                    <ColumnDefinition Width="*"/>
+                    <ColumnDefinition Width="auto"/>
                 </Grid.ColumnDefinitions>
 
-                <TextBlock Text="Installation Type:" Grid.Row="0" HorizontalAlignment="Left" VerticalAlignment="Center" Margin="5"/>
-                <ComboBox x:Name="InstallationTypeSelection" SelectedItem="{Binding InstallationType}" ItemsSource="{Binding AllInstallationTypes}" Grid.Row="1" Margin="0,0,0,0" HorizontalAlignment="Stretch" Height="NaN" VerticalAlignment="Stretch" Width="NaN" />
-                <CheckBox IsChecked="{Binding EnablePluginLogging }" Grid.Row="2" Content="Enable Plugin Logging" Margin="5,10,0,0" HorizontalAlignment="Left" Width="290"/>
-                <CheckBox IsChecked="{Binding EnablePluginAutomaticUpdates }" Grid.Row="3" Content="Enable Automatic Plugin Updates and Downloads" Margin="5,10,5,0"/>
-                <TextBlock Text="CLI Args:" Grid.Row="4" HorizontalAlignment="Left" VerticalAlignment="Top" Margin="5"/>
-                <TextBox Text="{Binding CLIArgs}" Grid.Row="5" TextWrapping="Wrap" IsManipulationEnabled="True" AcceptsReturn="True" />
-                <Button x:Name="CheckUpdateButton" Content="Check for updates" Grid.Row="6" Margin="5" Command="{Binding CheckForUpdateCommand}" />
+                <TextBlock Text="Installation Type:" Grid.Row="0" Grid.ColumnSpan ="2" HorizontalAlignment="Left" VerticalAlignment="Center" Margin="5"/>
+                <ComboBox SelectedItem="{Binding InstallationType}" ItemsSource="{Binding AllInstallationTypes}" Grid.Row="1" Grid.ColumnSpan ="2" Margin="0,0,0,0" HorizontalAlignment="Stretch" Height="NaN" VerticalAlignment="Stretch" Width="NaN" />
+                <CheckBox IsChecked="{Binding EnablePluginLogging }" Grid.Row="2" Grid.ColumnSpan ="2" Content="Enable Plugin Logging" Margin="5,10,0,0" HorizontalAlignment="Left" Width="290"/>
+                <CheckBox IsChecked="{Binding EnablePluginAutomaticUpdates }" Grid.Row="3" Grid.ColumnSpan ="2" Content="Enable Automatic Plugin Updates and Downloads" Margin="5,10,5,0"/>
+                <TextBlock Text="CLI Args:" Grid.Row="4" Grid.ColumnSpan ="2" HorizontalAlignment="Left" VerticalAlignment="Top" Margin="5"/>
+                <TextBox Text="{Binding CLIArgs}" Grid.Row="5" Grid.ColumnSpan ="2" TextWrapping="Wrap" IsManipulationEnabled="True" AcceptsReturn="True" />
+                <Button Content="Check for updates" Grid.Row="6" Grid.Column="0" Margin="5" Command="{Binding CheckForUpdateCommand}" />
```

Comment:
Removed unused names on these, and set colspan to 2, since there's 2 columns in this grid now to accommodate the version